ansible_collections/arista/avd/roles/eos_designs/templates/interface_descriptions/loopback_interfaces/overlay-loopback.j2

```diff
@@ -1 +1,10 @@
-{{ switch.overlay_loopback_description }}
+{% if overlay_loopback_description is not arista.avd.defined %}
```

Comment:
I am moving this logic away from the facts. Previously it was setting `switch.overlay_loopback_description` in the `switch.j2` which was then read here only. So with this I am moving the logic to the only place where it matters. This will also make it simpler to override.

Reply:
Ok so maybe the only comment based on our call would be to make the default part of the "default_values.yml" file rather than hardcoding them in the template here.

Reply:
This would be a larger initiative, hence not part of this refactoring.